jabel-javac-plugin/build.gradle.kts

```diff
@@ -0,0 +1,123 @@
+plugins {
+    java
+    `maven-publish`
+    signing
+}
+
+val jabelVersion: String by project
+version = jabelVersion
+
+dependencies {
+    implementation(platform("net.bytebuddy:byte-buddy-parent:1.14.9"))
+    implementation("net.bytebuddy:byte-buddy")
+    implementation("net.bytebuddy:byte-buddy-agent")
+    implementation("net.java.dev.jna:jna:5.13.0")
+}
+
+// <editor-fold desc="Publishing and Signing">
+
+tasks.withType<Javadoc>{
+    options.source = "8"
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = sourceCompatibility
+    withJavadocJar()
+    withSourcesJar()
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("mavenJava") {
+            artifactId = pomArtifactId
+            from(components["java"])
+            pom {
+                name.set(pomName)
+                packaging = pomPackaging
+                description.set(pomDescription)
+                url.set("https://github.com/pkware/jabel")
+                setPkwareOrganization()
+
+                developers {
+                    developer {
```

Comment:
If the original library had a developer that wasn't us, we should keep that. Again, we're not really doing anything here except publishing